ios/keyman/Keyman/Keyman/MainViewController.swift

```diff
@@ -1170,12 +1087,7 @@ class MainViewController: UIViewController, TextViewDelegate, UIActionSheetDeleg
     }
 
     let firstKB = userKbs[0]
-    let kbID = firstKB[Key.keyboardId] as? String
-    let langID = firstKB[Key.languageId] as? String
-    if (kbID == DefaultKeyboard.keyboardID) && (langID == DefaultKeyboard.languageID) {
-      return true
-    }
-    return false
+    return firstKB.id != Constants.defaultKeyboard.id || firstKB.languageID != Constants.defaultKeyboard.languageID
```

Comment:
I don't think this logic expression matches the original.  Shouldn't these be `==` and `&&` instead of `!=` and `||`?

Reply:
Yeah I don't know what happened here. :man_shrugging: